### ISSUE TITLE
Investigate why the windows binary fails in CI

### ISSUE THREAD

**Issue by flavorjones** [rubys/sprockets-esbuild#2]
See https://github.com/rubys/sprockets-esbuild/runs/4587785724?check_suite_focus=true for an example.

```
C:/hostedtoolcache/windows/Ruby/3.0.3/x64/lib/ruby/gems/3.0.0/gems/sprockets-esbuild-0.0.1-x64-mingw32/exe/esbuild:39:in `exec': Exec format error - C:/hostedtoolcache/windows/Ruby/3.0.3/x64/lib/ruby/gems/3.0.0/gems/sprockets-esbuild-0.0.1-x64-mingw32/exe/x64-mingw32/esbuild --help (Errno::ENOEXEC)
```

**Comment by flavorjones** [comment]
Ah, here's what's in the windows `esbuild` downloaded from npm:

```
#!/usr/bin/env node

// Unfortunately even though npm shims "bin" commands on Windows with auto-
// generated forwarding scripts, it doesn't strip the ".exe" from the file name
// first. So it's possible to publish executables via npm on all platforms
// except Windows. I consider this a npm bug.
//
// My workaround is to add this script as another layer of indirection. It'll
// be slower because node has to boot up just to shell out to the actual exe,
// but Windows is somewhat of a second-class platform to npm so it's the best
// I can do I think.
const esbuild_exe = require.resolve('esbuild-windows-64/esbuild.exe');
const child_process = require('child_process');
child_process.spawnSync(esbuild_exe, process.argv.slice(2), { stdio: 'inherit' });
```

**Comment by rubys** [comment]
```
>tar tzf esbuild-windows-64-0.14.5.tgz
package/bin/esbuild
package/esbuild.exe
package/package.json
package/README.md
```

I've tried the esbuild.exe in the package directory on Windows - it is the file we want.

**Comment by flavorjones** [comment]
Fixed by #3
